README.md

```diff
@@ -382,26 +383,26 @@ Examples
  - Connect to a simple remote shell (`socat` is required, VSOCK will be used):
 ```
    # Start the vng instance with server support:
-   $ vng --server
+   $ vng --console
 
    # In a separate terminal run the following command to connect to a remote shell:
-   $ vng --client
+   $ vng --console-client
 ```
 
  - Enable ssh in the vng guest:
 ```
    # Start the vng instance with ssh server support:
-   $ vng --server ssh
+   $ vng --ssh
```

Comment:
I'm not sure about introducing the `--scp` option, it doesn't seem to provide much compared to just running scp, it can help just to figure out the port, but I don't think it's worth to maintain an option just for that. I think we can simply show an example in the README on how to use scp and be happy with it. :)

Reply:
It makes sense. I suppose people interested in using `scp` will look at the `README` file, not just the man/`--help`.

In this case, no need to switch to `--ssh[d]` and `--console[d]` I suppose.